script/setup.js

```diff
@@ -275,7 +275,7 @@ try {
 			let user;
 			try {
 				user = JSON.parse((await $`gh api user`).stdout).login;
-			} catch (err) {
+			} catch (error) {
```

Comment:
Oh, heh, this is unused... +1 to calling it `error` over `err`. I suppose we should enable https://eslint.org/docs/latest/rules/no-unused-vars#caughterrors...

Reply:
#426